setup.py

```diff
@@ -70,7 +70,6 @@
     ],
     package_data={'pext': ['i18n/*.qm', 'images/scalable/*', 'qml/*', 'helpers/*', '*.py'],
                   'pext_dev': ['LICENSE', 'module/*', 'theme/*', '*.py']},
-    include_package_data=True,
```

Comment:
If you change this, please add VERSION to package_data for the pext package. pext/VERSION is currently defined in MANIFEST.in and therefore only included with include_package_data. I think MANIFEST.in can be deleted after that. (Interestingly enough, I think this may be the same thing that may fix #83)